src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Conditions/Condition.cs

```diff
@@ -15,29 +15,29 @@ public enum Condition
     {
         [DisplayName("Contains")]
         Contains,
-        [DisplayName("StartsWith")]
```

Comment:
That's already in Catel! https://github.com/Catel/Catel/blob/d90b2614ef1d67f8ac10065bda18c7a83ec78994/src/Catel.Core/Catel.Core.Shared/ComponentModel/DisplayNameAttribute.cs